### ISSUE TITLE
Crash on startup - Archlinx

### ISSUE THREAD

**Issue by Vinibo** [gnunn1/terminix#13]
When I try to start the application (installed through AUR script), I get this error:
```
(terminix:19777): GLib-CRITICAL **: g_variant_get_int64: assertion 'g_variant_is_of_type (value, G_VARIANT_TYPE_INT64)' failed
```
I'm on vanilla Gnome 3.18.2

**Comment by gnunn1** [comment]
I fixed a bug that was causing it to crash on Ubuntu, hopefully this fixes it for you as well. Give it a try and let me know if there is any change.

**Comment by Vinibo** [comment]
I had an update from AUR today, I don't have the crash anymore but doing terminix on terminal does nothing (prompt accepted without any error and that's all). I can't see any window or processes with the name "terminix".

Doing ldd doesn't show any missing links.

**Comment by gnunn1** [comment]
The application should be at /usr/bin/terminix, if you run it directly from another terminal does it just and quit with no info?

**Comment by Vinibo** [comment]
Yes exactly.
➜  ~  /usr/bin/terminix
➜  ~  echo $?          
0
I discovered that running terminix with sudo works.

**Comment by gnunn1** [comment]
That's weird, not sure what the issue is. Are you using bash or an alternative shell like zsh or fish? Also, are you using the gnome DE or something else?

**Comment by Vinibo** [comment]
You're right, I use ZSH as my login shell.
I tried 
```
export SHELL=/bin/bash terminix
```
but I have the same behavior than launching terminix from ZSH.
I also tried to launch bash and then launch terminix, same thing.
For the DE question, yes I use Gnome 3.18.2.
My $LANG for root and my user is fr_CA.UTF-8.

**Comment by gnunn1** [comment]
Merci beaucoup for the information, couldn't resist a little french to a fellow Canadian.

There is another issue filed for ZSH, #20 , so they may be related. My priority right now is fixing some memory issues and that will probably take me a couple of days, once that is done I'll look at this next.

**Comment by Vinibo** [comment]
I received an update this morning and it works and with the ZSH prompt!
Thanks!
